crates/fuel-gas-price-algorithm/src/lib.rs

```diff
@@ -4,3 +4,24 @@
 
 pub mod v0;
 pub mod v1;
+
+#[allow(clippy::cast_possible_truncation)]
+pub(crate) fn cumulative_percentage(
```

Comment:
I think we should name this "cumulative_percentage_change" or something like that instead

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2219/commits/adffd995e4949ba2e025d88ea6c51660187e1d7d